jasmine/spec/string-test.js

```diff
@@ -8,37 +8,55 @@ describe('toUpper', function() {
   it('should convert string to uppercase', function() {
     expect('small baby'.toUpper()).toEqual('SMALL BABY');
   });
+  it('should convert all alphabetic characters to uppercase', function() {
+    expect('Gamble**u'.toUpper()).toEqual('GAMBLE**U');
+  });
 });
 
 describe('toLower', function() {
   it('should convert all characters in string to lowercase', function() {
     expect('SMALL BABY'.toLower()).toEqual('small baby');
   });
+  it('should convert all alphabetic characters to lowercase', function() {
+    expect('GAMBLE**U'.toUpper()).toEqual('gamble**u');
+  });
 });
 
 describe('ucFirst', function() {
   it('should convert the first char of a string to uppercase', function() {
     expect('small baby'.ucFirst()).toEqual('Small baby');
+  });
+  it('should not alter the string at all', function() {
     expect('Kevin'.ucFirst()).toEqual('Kevin');
+  });
+  it('should not alter the first character of the string', function() {
     expect('?hi there'.ucFirst()).toEqual('?hi there');
   });
 });
 
 describe('isQuestion', function() {
-  it('should determine if a string is a question', function() {
-    expect('small baby'.isQuestion()).toBeFalsy();
+  it('should return true', function() {
     expect('small baby?'.isQuestion()).toBeTruthy();
   });
+  it('should return false', function() {
+    expect('How are you doing? I am doing great'.isQuestion()).toBeFalsy();
+    expect('small baby'.isQuestion()).toBeFalsy();
+  });
 });
 
-describe('words', function() {
-  it('should return an array of words in the string', function() {
-    expect('small baby'.words()).toEqual(['small', 'baby']);
-    expect('why oh why should you?'.words()).toEqual(['why', 'oh', 'should', 'you']);
+describe('words', function() {//add special characters
+  it('should return an array of words in the string', function() {//should be more modular
+    expect('babies are best'.words()).toEqual(['babies', 'are', 'best']);
+  });
+  it('should return an array of words without the special characters', function() {
```

Comment:
**Code Quality - Nitpick**

- the tests should be more modular to explain what they are testing